app/controllers/suggestions.rb

```diff
@@ -0,0 +1,73 @@
+get '/suggestions' do
+  @suggestion = Suggestion.all
+  @comment = Comment.all
```

Comment:
You don't need to get `Comment.all` since it would not be associated with a particular suggestion any way. Take advantage of activerecord and do something like `@suggestion.comments`

Also properly define your variables with plural